UnitTests/BraintreeCoreTests/Analytics/BTAnalyticsService_Tests.swift

```diff
@@ -19,7 +19,7 @@ final class BTAnalyticsService_Tests: XCTestCase {
         
         await analyticsService.performEventRequest("any.analytics.event")
         
-        XCTAssertEqual(analyticsService.http?.customBaseURL?.absoluteString, "https://api-m.paypal.com")
```

Comment:
When was this changed?

Reply:
PR: https://github.com/braintree/braintree_ios/pull/1175